code/parachain/frame/cosmwasm/src/ibc.rs

```diff
@@ -374,6 +374,8 @@ impl<T: Config + Send + Sync> IbcModule for Router<T> {
 		channel_id: &ChannelId,
 		counterparty: &Counterparty,
 		version: &IbcVersion,
+		_relayer: &IbcSigner,
+		// TODO: perhaps, we can remove this.
```

Comment:
probably best to refrain from a `TODO` here, if this could be improved maybe we can do that now or remove the `TODO`

Reply:
I read the issue, It looks important.
I'll remove `TODO`.
Perhaps, we should create a task for the issue.

Reply:
fixed.